agents/agents-core/src/commonMain/kotlin/ai/koog/agents/core/dsl/extension/AIAgentLLMActions.kt

```diff
@@ -24,6 +24,15 @@ public fun AIAgentLLMWriteSession.leaveLastNMessages(n: Int) {
     prompt = prompt.withMessages { it.takeLast(n) }
 }
 
+/**
+ * Removes the last `n` messages from the current prompt in the write session.
+ *
+ * @param n The number of messages to remove from the end of the current message list.
+ */
+public fun AIAgentLLMWriteSession.dropLastNMessages(n: Int) {
```

Comment:
@Ololoshechkin, if you prefer to keep the extension after comment from @tiginamaria below, what you think think about adding some tests here as well?